src/core/internal/dassert.d

```diff
@@ -191,6 +191,19 @@ private string miniFormat(V)(const scope ref V v)
     {
         return v ? "true" : "false";
     }
+    else static if (is(V == __vector(ET[N]), ET, size_t N))
+    {
+        string msg = "[";
+        foreach (i; 0 .. N)
+        {
+            if (i > 0)
+                msg ~= ", ";
+
+            msg ~= miniFormat(v[i]);
+        }
+        msg ~= "]";
+        return msg;
```

Comment:
Maybe forward the underlying array instead of copy+pasting the implementation:

```suggestion
        ET[] tmp = v.array;
        return miniFormat(tmp);
```

Reply:
I wanted to avoid the cut-off after 30 elements (a bit unfortunate for `ubyte32` etc.).